src/components/Microsoft.DotNet.UpgradeAssistant.Extensions/ExtensionOptions.cs

```diff
@@ -20,6 +22,10 @@ public class ExtensionOptions
 
         public Version CurrentVersion { get; set; } = null!;
 
+        public bool LoadExtensions { get; set; } = true;
+
         public string ConfigurationFilePath { get; } = Path.Combine(Environment.CurrentDirectory, "upgrade-assistant.json");
+
+        public string ExtensionCachePath { get; } = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData), "dotnet", "upgrade-assisistant", "extensions");
```

Comment:
What is a special folder?

Reply:
They're well-defined system folders. See https://docs.microsoft.com/en-us/dotnet/api/system.environment.specialfolder?view=net-5.0 for some details